CONTRIBUTING.md

```diff
@@ -145,60 +137,28 @@ All npm packages follow strict semantic versioning (or _semver_). Semantic versi
 
 — <https://semver.org/>
 
-**Note that we do not update package version numbers directly**, but instead depend on a [Conventional Commit](#conventional-commit) workflow which will version and release packages according to the content of our pull requests.
+**Note that we do not update package version numbers directly**, but instead depend on a [Changeset](#creating-a-changese) workflow which will version and release packages according to the content of changeset files.
 
-### Conventional Commit
+### Creating a Changeset
 
-Our pull requests need to be structured in a certain way for the CI pipeline to correctly update version numbers when releasing packages. By following the [release workflow outlined above](#release-workflow), our pull requests will result in a Conventional Commit to the main branch when merged (see the [Conventional Commit 1.0.0 spec](https://www.conventionalcommits.org/en/v1.0.0/)).
+Kaizen utilises [Changesets](https://github.com/changesets/changesets) to outline the changes made to specific packages as well as keeps all depenencies of the changed packages up to date.
 
-This workflow requires that all pull requests have a title formatted as follows:
+Creating a changeset is straight forward:
 
-```
-<type>: <description>
-```
+1. Run `yarn changeset` (or `yarn commit`)
+2. Select the packages you have made changes to with spacebar, followed by enter
+3. You will then be asked which packages should have a Major, Minor or Patch. So following [Semantic Versioning](#semantic-versioning), select the packages that apply and hit enter (you can also hit enter to skip)
+4. Add a short [description](#description).
+
+A changeset `md` file will be added to your branch which you are free to modify the details of if necessary.
 
-To help you, we have [`commitizen`](https://github.com/commitizen/cz-cli) installed, so you can run `yarn commit` and follow the prompts and commitizen will format your commit for you.
 
 #### Description
 
 The `description` should be a short (less than ~60 characters) summary of changes introduced in the release. This summary will be included in the CHANGELOG (see the [releases page](https://github.com/cultureamp/kaizen-design-system/releases)), along with the commit type, a link to the pull request which triggered the release, and a detailed summary of any breaking changes.
 
 Note that since the `description` will be included in the CHANGELOG — and may be the only summary of the changes that your colleagues will encounter — it is helpful for you to include a concise summary of _how the package is different_ following the release, written in the [imperative mood](https://chris.beams.io/posts/git-commit/#imperative). For example `fix: Address accessibility bug in the Gizmo component`.
 
```

Comment:
Probably not needed just yet but wouldn't mind providing guidance / a loose structure on what makes a good changeset description - once we figure out what that is 😆